src/Stratis.StratisDnsD/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.Protocol;
+using Stratis.Bitcoin;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Dns;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.StratisDnsD
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            MainAsync(args).Wait();
+        }
+
+        public static async Task MainAsync(string[] args)
+        {
+            try
+            {
+                Network network = args.Contains("-testnet") ? Network.StratisTest : Network.StratisMain;
```

Comment:
The network shouldn't be limited to Stratis.
Have a look in BreezeD, there we support stratis or bitcoin: https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/Stratis.BreezeD/Program.cs#L34.
So for stratis you'd do `dotnet run stratis -testnet` and bitcoin you'd do `dotnet run -testnet`.

Reply:
Just realized that this project is called Stratis.StratisDnsD so maybe your intention was indeed to create a stratis only project.
How about making it network agnostic and calling it Stratis.DnsD?

Reply:
I have just checked and I believe the decision on Monday was to limit this to only the Stratis network. The PR was created with this in mind.

Reply:
Ok cool mate, that's fair enough.